File: hyperion-appinfo/src/main/java/com/star_zero/hyperion/appinfo/AppInfoPlugin.java
package com.star_zero.hyperion.appinfo;

import androidx.annotation.Nullable;

import com.google.auto.service.AutoService;
import com.willowtreeapps.hyperion.plugin.v1.Plugin;
import com.willowtreeapps.hyperion.plugin.v1.PluginModule;

@AutoService(Plugin.class)
public class AppInfoPlugin extends Plugin {
    @Nullable
    @Override
    public PluginModule createPluginModule() {
        return new AppInfoModule();
    }
}

File: hyperion-appinfo/src/main/java/com/star_zero/hyperion/appinfo/AppInfoPlugin.kt
package com.star_zero.hyperion.appinfo

import com.google.auto.service.AutoService
import com.willowtreeapps.hyperion.plugin.v1.Plugin
import com.willowtreeapps.hyperion.plugin.v1.PluginModule

@AutoService(Plugin::class)
class AppInfoPlugin : Plugin() {
    override fun createPluginModule(): PluginModule {
        return AppInfoModule()
    }
}